wordpress-mobile/WordPress-iOS
# Signup NUX: new prologue

Refs #8109 

This adds the new sign up buttons to the login prologue.

![prologue methods](https://user-images.githubusercontent.com/517257/36188564-668c28fa-1112-11e8-8a36-78c7631fccc7.gif)

Notes:
- Signup with Google will be a future PR
- Uses the same `NUXButtonsView` as other parts of the new NUX
  - changed to allow top and bottom button to controller as white/blue 
  - changed buttons to have minimum 50px height. This is what the prologue buttons were before, but we may want @folletto to ok making them all match

To test:
- Try to the new prologue "Sign up" UI
- Make sure the old flow works with the feature flag off
